src/arch/arm/pagetable.hh

```diff
@@ -162,7 +163,7 @@ struct V8PageTableOps64k : public PageTableOps
 };
 
 // ITB/DTB table entry
-struct TlbEntry : public Serializable
+struct TlbEntry : public ReplaceableEntry, Serializable
```

Comment:
Minor nit: can we please rename 'N' to be a bit more descriptive like 'pageSizeBits' or similar?

Reply:
As other comments. This makes a lot of sense but it is something historical and I would open a new PR for that, so that this one keeps being "easy" to review